SETUP.md

```diff
@@ -64,6 +64,7 @@ On Mac OS, _SBT_ provides instructions [here](https://www.scala-sbt.org/1.x/docs
 ### Update your kubeconfig
 
 ```
+aws configure   # requires AWS Access Key(s)
```

Comment:
This paragraph intentionally is about the kubernetes configuration only, not about a configuration of other tools listed here. 

In example `sbt` also has to be configured for this flow to run, but it's omitted here and explained in the [other place](https://github.com/Constellation-Labs/tessellation/blob/develop/CONTRIBUTING.md#setting-up-sbt).

If you wish to explain how to configure AWS CLI, please add a separate paragraph about it.

Reply:
I added the AWS section. I don't understand the comment about `sbt` though.

Reply:
@sdavidp in order to use skaffold, you need to configure the sbt first (to compile the project). it's ommited in the skaffold documentation as it's specified in other place (how to configure your dev env)